chipy_org/apps/sponsors/models.py

```diff
@@ -39,6 +39,14 @@ class Meta:
         ordering = ["sponsor__name"]
 
 
+class SponsorGroup(models.Model):
+    name = models.CharField(max_length=80)
+    list_priority = models.IntegerField(default=5)
+
+    def __str__(self):
+        return self.name
+
+
```

Comment:
Ray, I wonder if a [choice field](https://docs.djangoproject.com/en/2.2/ref/models/fields/#choices) is more appropriate than a very flexible model.

My concern is someone could either a) author a similar group or b) create a SponsorGroup with the same list_priority.  

The flexibility is nice to have, but maybe unneeded if you already have well defined groups in mind.  

The counter argument is adding a new SponsorGroup would not require a deploy.

**Approved**